source/dbo/Views/Power BI Reporting/vwMissingHospitalMappings.sql

```diff
@@ -0,0 +1,61 @@
+﻿CREATE VIEW [dbo].[vwMissingHospitalMappings]
+AS
+
+-- LTBR hospital is not in migration mapping
+SELECT CONCAT(dp_PatientID, '-', dp_DiseasePeriod) AS OldNotificationId,
+	'LTBR' AS [Source],
+	dp_NotifiedDate AS NotificationDate,
+	CAST(dp_NotifyingClinicID AS NVARCHAR(50)) AS HospitalId,
+	lh_HospitalName COLLATE Latin1_General_CI_AS AS HospitalName,
+	residencePhec.[Name] AS ResidenceRegion,
+	CAST(NULL AS NVARCHAR) AS TreatmentRegion
```

Comment:
I couldn't think of a good way of trying to extract this from LTBR, if anyone can think of a way do let me know otherwise they can just be left as `NULL`